Kudu.Services.Web/Pages/NewUI/_Layout.cshtml

```diff
@@ -69,49 +69,54 @@
   <div class="row">
```

Comment:
I think we can remove `Browse wwwroot`, `Browse LogFiles` from the App Essentials tab in `Index.xshtml`